scripts/parse.js

```diff
@@ -8,22 +8,20 @@ const {
   writeJson
 } = require('./shared')
 
-fs.readdir(CHAINS_DIRECTORY, async function (err, files) {
-  if (err) {
-    console.error('Could not list the directory.', err)
-    process.exit(1)
-  }
-
-  startSpinner(`Verifying ${CHAINS_DIRECTORY} files`)
-
+async function processFiles (files) {
   await Promise.all(
-    files.map(async function (file, index) {
+    files.map(async function (file) {
       const filePath = path.join(CHAINS_DIRECTORY, file)
       const fileStat = await stat(filePath)
       const ext = path.extname(file)
       if (fileStat.isFile() && ext === '.json') {
         let json = require(filePath)
 
+        let fileName = file.replace('.json', '')
```

Comment:
can we use `file.replace(/\.json$/, '')` to ensure we only replace at filename end?

Reply:
Done 👍